chapter02/mssw/exercise1.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Cwd;
+use File::Spec;
+
+my $cwd = getcwd();
+
+my $thing = File::Spec->catfile($cwd);
+
+print "How exactly is File::Spec and Cwd supposed to help list directories?!?!\n";
```

Comment:
They help you get the full path to a file, not the directory listing. Read the question harder.